packages/angular/src/builders/module-federation-dev-server/module-federation-dev-server.impl.ts

```diff
@@ -48,9 +51,24 @@ export function executeModuleFederationDevServerBuilder(
 
   validateDevRemotes(options, workspaceProjects);
 
+  const patternMatchedRemotesToSkip = new Set(
+    findMatchingProjects(options.skipRemotes, projectGraph.nodes)
+  );
+  const remotesNotInGraph = options.skipRemotes.filter((r) => {
+    const { exclude, type } = parseStringPattern(r, projectGraph.nodes);
+    return exclude === false && type === 'unlabeled';
+  });
   const remotesToSkip = new Set(
-    findMatchingProjects(options.skipRemotes, projectGraph.nodes) ?? []
+    [...patternMatchedRemotesToSkip, ...remotesNotInGraph] ?? []
   );
+
+  if (remotesToSkip.size > 0) {
+    console.log(
```

Comment:
Let's use the Nx `logger`:
```suggestion
    logger.info(
```